ReactNative/ReactTwitter/core/views/splash-screen.js

```diff
@@ -49,7 +50,7 @@ var SplashScreenView = React.createClass({
   render () {
     return (
       <View style={styles.container}>
-        <Text style={styles.title}>SplashScreen</Text>
+        <Image style={styles.logo} source={require('./assets/twitter-logo.png')} />
```

Comment:
Loading it like that, is it going to work for both platforms? (I guess so)
It would be good to see how to include assets for different screen densities. In android we have this problem, how is it in iOS? (different screens?)